plugin/src/main/groovy/net/madeng/gradle/plugin/slack/api/Action.groovy

```diff
@@ -0,0 +1,31 @@
+package net.madeng.gradle.plugin.slack.api
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+class Action {
+
+  Confirm confirm
+  String name;
+  String text;
+  String style;
+  String type;
+  String value;
+
+  def confirm(Closure closure) {
+    closure.resolveStrategy = Closure.DELEGATE_FIRST
+    confirm = new Confirm()
+    closure.delegate = confirm
+    closure()
+  }
+
+  class Confirm {
+    String title;
+    String text;
+    String ok_text;
```

Comment:
I really appreciate your contribution. Your PR looks good to me but I have only a small request for keeping fields' format. Can you change `ok_test` to `okTest` and `dismiss_text` to `dismissText`?